main/ertp/guide/README.md

```diff
@@ -26,87 +26,14 @@ post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
 ERTP is the top layer of the Agoric stack. You can find out more about this stack in this introductory video:
 <iframe width="560" height="315" src="https://www.youtube.com/embed/52SgGFpWjsY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## A quick tutorial
 
-Let's look at an example. In ERTP, all digital assets, including fungible and
-non-fungible tokens, are created by a [`mint`](./mint.html). Having access to the mint
-gives you the power to create more digital assets of the same type at
-will. For instance, let's say we want to create a new community
-currency called 'BaytownBucks'.
-You would first install the [ertp JavaScript package](https://www.npmjs.com/package/@agoric/ertp)
-(`npm install @agoric/ertp`) and then:
+## Guide
 
-```js
-import { makeMint } from '@agoric/ertp/core/mint';
+Different resources can help you understand what can be done with ERTP, but also how to do it:
```

Comment:
Can we change the language to something like: "Let's get started by creating some different kinds of digital assets." I'm afraid that "Different resources can help you..." is too vague

Reply:
:+1: